data/sql/updates/pending_db_world/rev_1679482307407253900.sql

```diff
@@ -0,0 +1,3 @@
+--
+-- Remove Pollution from Water Barrels (gameobject 3658)
+DELETE FROM gameobject_loot_template WHERE Entry=2502 AND Item IN(851, 852, 853, 854, 858, 1196, 1197, 1198, 2207, 2455, 4765, 4766, 4777, 4778);
```

Comment:
backticks and a space after IN
```suggestion
DELETE FROM `gameobject_loot_template` WHERE `Entry`=2502 AND `Item` IN (851, 852, 853, 854, 858, 1196, 1197, 1198, 2207, 2455, 4765, 4766, 4777, 4778);
```